### PR TITLE
Add initial support for kotlin

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

**Comment by lburgazzoli** [comment]
ok to test

File: extensions/kotlin/runtime/src/main/kotlin/org/apache/camel/quarkus/kotlin/RoutesDSL.kt [hunk 1]
```diff
@@ -14,14 +14,15 @@
...
 
+import org.apache.camel.RoutesBuilder
 import org.apache.camel.builder.RouteBuilder
 
-class TimerRoute: RouteBuilder() {
-    override fun configure() {
-        from("timer:foo?period=1s")
-                .process { e -> e.message.body = "Hello from Kotlin!" }
-                .log("\${body}")
+fun routes(block: RouteBuilder.() -> Unit) : RoutesBuilder {
```
Comment (ppalaga) [inline extensions/kotlin/runtime/src/main/kotlin/org/apache/camel/quarkus/kotlin/RoutesDSL.kt]: Could you please add a piece of KDoc explaining what is the code in this file supposed to do?

Comment (lburgazzoli) [reply extensions/kotlin/runtime/src/main/kotlin/org/apache/camel/quarkus/kotlin/RoutesDSL.kt]: it's fairly obvious but I'll add some doc

Comment (ppalaga) [reply extensions/kotlin/runtime/src/main/kotlin/org/apache/camel/quarkus/kotlin/RoutesDSL.kt]: > it's fairly obvious but I'll add some doc

Do not laugh at us - Kotlin noobs!

**Comment by lburgazzoli** [comment]
ok to test

**Pull request merged**